pulser-core/pulser/_seq_drawer.py

```diff
@@ -154,6 +174,16 @@ def phase_str(phi: float) -> str:
     data = gather_data(seq)
     total_duration = data["total_duration"]
     time_scale = 1e3 if total_duration > 1e4 else 1
+    curves_per_ch = {}
+    for ch in seq._schedule:
+        curves_per_ch[ch] = [True, False, False]
+        if np.nonzero(data[ch]["detuning"])[0].size > 0:
+            curves_per_ch[ch][1] = True
+        if draw_phase_curve and np.nonzero(data[ch]["phase"])[0].size > 0:
+            curves_per_ch[ch][2] = True
```

Comment:
Here, the correspondence between the indices and "detuning/phase" seems magic. It might be good to have it at one common place.